build.sbt

```diff
@@ -48,6 +53,17 @@ lazy val dottyTensorflow = project
   )
   .dependsOn(scalaPyTensorFlowJVM)
 
+lazy val dottyTensorFlowExamples = project
```

Comment:
Hmm weird but this still looks published in `sbt publishM2`:

![image](https://user-images.githubusercontent.com/3383210/88297090-cb6f0980-ccff-11ea-9d3e-c06181df2181.png)

Possibly yet another setting is needed to disable local Maven publish (as opposed to the default, Ivy publish)

Reply:
Note that if want to serve this repo over jitpack, then jitpack runs `sbt publishM2` indeed